src/main/java/org/interledger/setup/SetupService.java

```diff
@@ -0,0 +1,40 @@
+package org.interledger.setup;
+
+import javax.money.MonetaryAmount;
+
+import org.interledger.ilp.InterledgerPaymentRequest;
+import org.interledger.setup.model.Receiver;
+import org.interledger.setup.model.ReceiverQuery;
+
+
+public interface SetupService {
+
+  /**
+   * The sender (this service) queries the receiver endpoint to get information about the type of 
+   * payment that can be made to this receiver:
+   *
+   * @param receiverEndpoint
```

Comment:
Will do. As with the Exceptions. JavaDoc needs a complete review. Will put that on my TODO list.